### PR TITLE
DEVOPS-475 Initial changes to remove requirement for setuid

### PR DESCRIPTION
**Pull request by sychan**
# Description of PR purpose/changes

* Change file ownership to remove any need for the setuid capability in order to write config.json at startup.
* It is possible to remove the capability to setuid for processes in a docker container. This is considered a best practice for securely running docker images. 

# Jira Ticket / Issue #
https://kbase-jira.atlassian.net/browse/DEVOPS-475
- [x] Added the Jira Ticket to the title of the PR (e.g. `DATAUP-69 Adds a PR template`)

# Testing Instructions
* Details for how to test the PR:
- [ ] Tests pass locally and in GitHub Actions
- [x] Changes available by spinning up a local narrative and navigating to _X_ to see _Y_

# Dev Checklist:

- [ ] My code follows the guidelines at https://sites.google.com/lbl.gov/trussresources/home?authuser=0
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] (JavaScript) I have run Prettier and ESLint on changed code manually or with a git precommit hook
- [ ] (Python) I have run Black and Flake8 on changed Python code manually or with a git precommit hook

# Updating Version and Release Notes (if applicable)

- [ ] [Version has been bumped](https://semver.org/) for each release
- [x] [Release notes](/RELEASE_NOTES.md) have been updated for each release (and during the merge of feature branches)

### CODE REVIEW

**Review by briehl — APPROVED** [review]
I think this looks fine. There's an image built, should we just deploy it on CI and see? Or merge first and back out if it fails? Were you able to run it locally?

**Comment by sychan** [comment]
I tested it locally and it seemed to work fine, I was waiting for a few more test fix PRs to merge them in an retry the automated tests. After merging in tests today and retrying some tests that timed out, it seems to pass. So I'm game for testing in Next and then CI.

**Comment by briehl** [comment]
Great. Let's merge, then, and give it a go.

**Pull request merged**
